ISSUE_TEMPLATE/doc_issue.md

```diff
@@ -0,0 +1,34 @@
+### Does the doc defect  apply to a code bug? 
+- Yes __
+  - Specify the issue for the code bug ____.
+  - Add the Doc bug label.
+- No __
+
+### Does the documentation apply to a particular release?
+- Yes ____
+  - Specify the release:____
+- No  ____
+
+### Does the documentation apply to all operating systems? 
+- Yes __  
+- No  __ 
+  - Specify operating systems: ______
+
+### Specify what you want documented (required):
```

Comment:
shouldn't this be the first thing we ask?

Reply:
We ask whether the issue is for a bug and to what release it belongs first so that the author is more likely to provide this information. We use a similar strategy for other templates, and it works.

Reply:
The template doesn't ask what release it belongs to and it doesn't ask if it is an bug in the documentation either.

Reply:
I'm not following your comment. I see "Does the doc update apply to a doc bug? " and "Does the documentation apply to a particular release?". So, the template asks if the update is for a doc bug, and tells the author to specify the release.